app/forms/reset_password_form.rb

```diff
@@ -6,13 +6,15 @@ class ResetPasswordForm
 
   validate :valid_token
 
-  def initialize(user)
+  def initialize(user, options = {})
```

Comment:
Does this need to be configurable? What happens if we don't make this an option and just always validate?